managed-services/transactional-email/concepts.mdx

```diff
@@ -0,0 +1,195 @@
+---
+meta:
+  title: Transactional Email - Concepts
+  description: Learn the fundamental concepts of Scaleway Transactional Email.
+content:
+  h1: Transactional Email - Concepts
+  paragraph: Learn the fundamental concepts of Scaleway Transactional Email.
+tags: managed-services deliverability dkim dns-propagation mx-record send-api smtp transactional spf
+dates:
+  validation: 2024-11-04
+categories:
+  - managed-services
+---
+
+## Aggregate report
+
+An aggregate report (`rua`) is a periodic summary that provides data about a domain's email authentication activity. It includes information such as the volume of emails received, the percentage of emails that pass or fail DMARC checks, and the sources of failed authentication.
+
+## Blocklist
+
+Blocklist allows your to manage email addresses that generate errors, or “bounces”. You can block them improve the quality of your email deliveries, maintain your domain's reputation score, and reduce the risk of having your domains blocked by ISPs.
+
+## Deliverability
+
+Deliverability is the ability for an email to arrive in a recipient’s email inbox.
+
+## DKIM record
+
+A Domain Keys Identified Mail (DKIM) record is an email security standard. It provides you with an encryption key and a digital signature that allows you to verify whether an email was altered.
+
+## DMARC record
+
+DMARC stands for **D**omain-bassed **M**essage **A**uthentication **R**eporting and **C**onformance. It is an email authentication protocol that helps prevent email spoofing and phishing attacks by verifying that the sender's domain is legitimate.
+DMARC builds upon two other email authentication protocols: [SPF](#spf-record) and [DKIM](#dkim-record). It allows domain owners to publish policies in their Domain Name System (DNS) records specifying how emails sent from their domain should be handled if they fail authentication checks.
```

Comment:
```suggestion
DMARC builds upon other email authentication protocols: [SPF](#spf-record) and [DKIM](#dkim-record). It allows domain owners to publish policies in their Domain Name System (DNS) records specifying how emails sent from their domain should be handled if they fail authentication checks.
```

I think removing the number helps for future updates as we will forget to update this number if a new protocol is required to domain creation.